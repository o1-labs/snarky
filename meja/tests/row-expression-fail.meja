let x : [< `A | `B | `C > `A] = `D;
