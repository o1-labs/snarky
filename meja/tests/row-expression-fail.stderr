File "tests/row-expression-fail.meja", line 1, characters 32-34:
Error: Incompatible types ([<`A | `B | `C > `A] as 'a) and ([>`D] as 'b):
       Cannot unify ([<`A | `B | `C > `A] as 'a) and ([>`D] as 'b):
       the constructor `D is not present in both.
